src/main/java/exception/DukeUnknownException.java

```diff
@@ -0,0 +1,8 @@
+package exception;
+
+public class DukeUnknownException extends DukeException {
+    @Override
+    public String getMessage() {
+        return "OOPS!!! I'm sorry, but I don't know what that means :-(";
```

Comment:
not sure whether too overly done if instead of the String here, replace it with something like: 
`Ui.feedbackUnknownMessage()`, maybe a static method for simplicity